redi-connect-front/src/components/organisms/Footer.tsx

```diff
@@ -1,83 +1,95 @@
 import React from 'react'
-import { Columns, Container, Section } from 'react-bulma-components'
+import { Element, Columns, Container, Section, Heading } from 'react-bulma-components'
 import { useTranslation } from 'react-i18next'
+import { ReactComponent as RediSchool } from '../../assets/images/redi-school-logo.svg'
+import { ReactComponent as Deloitte } from '../../assets/images/deloitte.svg'
 import Icons from '../atoms/MediaIcons'
 import './Footer.scss'
 
 const RediFooter = () => {
   const year = new Date().getFullYear()
   const { t } = useTranslation()
 
+  const supportLinks = [
+    {
+      url: 'https://connect.redi-school.org/downloadeables/redi-connect-code-of-conduct.pdf',
+      name: t('footer.faq')
+    },
+    {
+      url: 'mailto:career@redi-school.org',
+      name: t('footer.contact')
+    },
+    {
+      url: 'https://www.redi-school.org',
+      name: t('footer.name')
+    }
+  ]
+
+  const legalLinks = [
+    {
+      url: 'https://www.redi-school.org/berlin-transparency/',
+      name: t('footer.transparency')
+    },
+    {
+      url: 'https://www.redi-school.org/imprint/',
+      name: t('footer.cookies')
+    },
+    {
+      url: 'https://www.redi-school.org/data-privacy-policy',
+      name: t('footer.privacy')
+    }
+  ]
+
   return (
-    <footer className="footer">
+    <footer className="footer footer--border">
       <Section>
         <Container>
-          <Columns>
-            <Columns.Column className="is-hidden-mobile">
-              <p>
-                <a href="https://www.redi-school.org/">{t('footer.name')}</a>
-              </p>
-              <p>&copy; {year} {t('footer.copyright')}</p>
+          <Columns breakpoint="mobile">
+            <Columns.Column mobile={{ size: 'full' }}>
+              <Element>
+                <RediSchool className="oneandhalf-bs" />
+              </Element>
+              <Element>
+                <Deloitte />
+              </Element>
             </Columns.Column>
-            <Columns.Column className="is-hidden-tablet footer--headline">
-              <p className="is-size-5">{t('footer.socialMediaHeadline')}</p>
-              <Icons />
+            <Columns.Column mobile={{ offset: 'zero', size: 'half' }} desktop={{ offset: 1 }} className="double-bs">
+              <Heading subtitle size={6} textTransform="uppercase" textWeight="bold">
+                {t('footer.support')}
+              </Heading>
+              <Element renderAs="hr"/>
+              {supportLinks.map(link => (
+                <Heading subtitle size={6} textTransform="uppercase">
```

Comment:
SEO Department (if one existed) would most possibly be pretty offended by this. 
This is actually rendering H1 tags. So please ALWAYS check the outcome in the browser.

Other than that, the component is completely wrong here. A correct semantic would be a list here.

Reply:
refactored with list tag